apps/target_manager.py

```diff
@@ -105,6 +106,16 @@ def create_target(targets_json, compose_apps, ota_lite_tag, git_sha, machines, p
             tgt_name = tagmgr.create_target_name(target, version, tag)
             data['targets'][tgt_name] = target
 
+            now = datetime.datetime.utcnow().strftime('%Y-%m-%dT%H:%M:%SZ')
+            target['custom']['createdAt'] = now
```

Comment:
iiuc, it is used to be inherited from LmP build?

Reply:
In this case, we overwrite metadata about an ostree Target which is part of an overall Target.

Reply:
> it is used to be inherited from LmP build?

correct. This sort of makes a linked-list via the origUri to find the actual LmP build now.

> In this case, we overwrite metadata about an ostree Target which is part of an overall Target.

we could preserve the original createdAt field like we do with origUri, but I was worried that's started to get a little verbose and we can deduce the original createdAt timestamp based on the other data we have.